src/deploy-cromwell-on-azure/scripts/helm/values-template.yaml

```diff
@@ -61,7 +61,6 @@ images:
 # Mounted as /containername as expected by Cromwell
 cromwellContainers:
   - configuration
-  - cromwell-executions
   - cromwell-workflow-logs
```

Comment:
With the change in `workflow-options`, should this line also be removed? Or is it still necessary?

Reply:
Doesn't seem to work so I'm reverting the logs portion for now.